packages/librsvg.rb

```diff
@@ -25,7 +25,12 @@ def self.build
     system "rustup install stable --profile minimal || (rm -frv ~/.rustup/toolchains/* && rustup install stable --profile minimal)"
     system "rustup default stable"
     system "./configure",
-      #{CREW_OPTIONS},
+      "--prefix=#{CREW_PREFIX}",
+      "--libdir=#{CREW_LIB_PREFIX}",
+      "--mandir=#{CREW_MAN_PREFIX}",
+      "--build=#{CREW_BUILD}",
+      "--host=#{CREW_BUILD}",
+      "--target=#{CREW_BUILD}",
       "--enable-introspection=yes",
       "--enable-vala=yes",
       "--disable-static",
```

Comment:
Try this instead:
```
system "./configure #{CREW_OPTIONS} --enable-introspection=yes --enable-vala=yes --disable-static"
```

Reply:
The problem is with the way ruby parses strings separated by commas.  `CREW_OPTIONS` is space delimited.

Reply:
Right, but if you have CREW_OPTIONS  on a different line, then the other lines are ignored, and so is CREW/MESON_OPTIONS.

So all following options have to be space delimited on that line?

Reply:
Yes, unfortunately the ruby parser gets confused when it builds the command if you mix and match space and comma delimited arguments.